src/quo2/components/inputs/title_input/view.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.inputs.title-input.view
+  (:require
+    [quo2.components.inputs.title-input.style :as style]
+    [reagent.core :as reagent]
+    [quo2.components.markdown.text :as text]
+    [react-native.core :as rn]))
+
+(defn- pad-0
+  [value]
+  (if (<= (count value) 1)
+    (str 0 value)
+    value))
+
+(defn title-input
+  [{:keys [on-change-text
+           placeholder
+           max-length]
+    :or   {max-length 0}}]
+  (let [focused? (reagent/atom false)]
+    (fn [{:keys [disabled? blur?]} value]
+      [rn/view
+       {:style style/container}
+       [rn/view
+        [rn/text-input
```

Comment:
The cursor's color is different when we change the theme and blur variants:
(blue, black and white, for example):
![image](https://user-images.githubusercontent.com/90291778/220447748-b0449275-29c5-475e-97a3-60e70692021a.png)

Reply:
I've seen some problems while testing on Android: Pixel4 API 30:

1. 
![image](https://user-images.githubusercontent.com/90291778/220449115-caf9ef08-1aab-443b-b77a-1f27837ee445.png)
the sentence is incomplete but there is still room for it to fit.

2. It seems "00/24" is a little lower than the designs, but IDK whether this is related to the implementation or  phone's font size
![image](https://user-images.githubusercontent.com/90291778/220449536-f9a30a5c-e921-4de4-a8a6-10a312b91484.png)
![image](https://user-images.githubusercontent.com/90291778/220449606-44aea5ea-6f5b-470d-a4bd-b4ad8000153c.png)

3. A strange "." appears in the placeholder after you write:
[Screencast from 2023-02-21 14-22-57.webm](https://user-images.githubusercontent.com/90291778/220450358-0eb918d4-3f1c-4dfb-bc81-021bd2d26238.webm)

Reply:
I will take a look, think I need to re-approach that alignment. Will adjust the cursor too!